### PR TITLE
Font updates in new filters

### PR DESCRIPTION
**Pull request by yosilevy**
## Proposed change

Font updates in new filters

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-label.ts
Comment (yosilevy) [reply src/components/ha-label.ts:47]: I'll revert it, especially since I can't find the place it didn't work in... :-) But I will fix ha-input-chip instead. That definitely needs it and it gets Arial if not using their font CSS variable.

Comment (yosilevy) [reply src/components/ha-label.ts]: @bramkragten I reverted it. Waiting for your approval.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**
